src/disputables/index.js

```diff
@@ -25,10 +25,18 @@ const DELAY_APP_ID =
   '0x133c97c74d2197a068988549d31108ce57af8f0ccf90ff9edf0ba5d349f2a450' // disputable-delay.aragonpm.eth
 const VOTING_APP_ID =
   '0x09cdc3e6887a0002b11992e954a40326a511a1750a2f5c69d17b8b660b0d337a' // disputable-voting.aragonpm.eth
-// '0x705b5084c67966bb8e4640b28bab7a1e51e03d209d84e3a04d2a4f7415f93b34' // disputable-voting.open.aragonpm.eth
+const VOTING_OPEN_APP_ID =
+  '0x705b5084c67966bb8e4640b28bab7a1e51e03d209d84e3a04d2a4f7415f93b34' // disputable-voting.open.aragonpm.eth
```

Comment:
Added support for the `open` version as well of disputable voting since that's where Facu was testing.